src/main.js

```diff
@@ -109,7 +109,8 @@ function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 }
 
-
+function createRandomImage() {
+var poster = document.querySelector('.poster');
 var randomImage = images[getRandomIndex(images)];
-
-console.log(randomImage);
\ No newline at end of file
+poster.innerHTML = `<img src="${randomImage}">`
```

Comment:
I see where you're going. Let's review this line again and further documentation about how to use .innerHTML